web_duplicate_visibility/demo/res_users_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+     <record id="view_users_form" model="ir.ui.view">
+        <field name="inherit_id" ref="base.view_users_form"/>
+        <field name="name">res.users.form</field>
```

Comment:
FYI, this line is useless. if this field is not here, odoo will generate automatically the name as 'model.name' + 'view.type'. here : res.users.form